index.html

```diff
@@ -523,6 +531,116 @@ <h4>Workflow Type</h4>
             </pre>
           </div>
         </section>
+
+        <section>
+          <h4>Default Language</h4>
+          <p>The <dfn>Default Language</dfn> is a mandatory property of a <a>DAPT Script</a>
+            which represents the default language for the <a>Text</a> content of <a>Script Events</a>.
+            This language may be one of the <a>Original Languages</a> or a <a>Translation</a> language.
+            When it represents a <a>Translation</a> language, it may be the final language
+            for which a dubbing or audio description <a>script</a> is being prepared,
+            called the <dfn>Target Recording Language</dfn> or it may be an intermediate, or pivot, language
+            used in the workflow.
+          </p>
+
+          <p>The <a>Default Language</a> is represented in a <a>DAPT Document</a> by the following structure and constraints:
+            <ul>
+              <li>the <code>xml:lang</code> attribute MUST be present on the <code>&lt;tt&gt;</code> element and its value MUST NOT be empty.</li>
```

Comment:
This is an important restriction of TTML2. It needs to be documented in the feature list in the appendix but I am not to sure which feature needs to be commented

Reply:
It would be `#xmlLang-root`.

Reply:
Done in 74e4017